src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -123,14 +130,47 @@ def __init__(self, **kwargs):
         self.xy_move_widget = widget_xy_move_drywall.XYMoveDrywall(machine=self.m, screen_manager=self.sm)
         self.xy_move_container.add_widget(self.xy_move_widget)
 
+        self.drywall_shape_display_widget = widget_drywall_shape_display.DrywallShapeDisplay(machine=self.m, screen_manager=self.sm, dwt_config=self.dwt_config)
+        self.shape_display_container.add_widget(self.drywall_shape_display_widget)
+
+        self.shape_selection.text = 'circle'
+
     def home(self):
         self.m.request_homing_procedure('drywall_cutter', 'drywall_cutter')
 
     def select_shape(self):
-        pass
-
-    def rotate_shape(self):
-        pass
+        if self.shape_selection.text in ['line', 'geberit']:
+            # Only on line available for these options
+            self.cut_offset_selection.text = 'on'
+            self.cut_offset_selection.disabled = True
+        else:
+            # Default to cut inside line
+            self.cut_offset_selection.text = 'inside'
```

Comment:
Check this behaviour with Az or Ed - I don't know if the cut offset should revert to being inside every time the shape is changed (instead maybe it should follow whatever the prev shape had?). Can be done as a later ticket if it is a thing though :)

Reply:
Default offset was specified [here](https://docs.google.com/spreadsheets/d/1X37CWF8bsXeC0dY-HsbwBu_QR6N510V-5aPTnxwIR6I/edit#gid=0)